site/layouts/blog/single.html

```diff
@@ -66,6 +66,14 @@ <h6>Other Posts</h6>
                         {{ end }}
                         {{ end }}
                     </ul>
+                     <h6>Categories</h6>
+                    <ul class="list-unstyled">
+                    <ul class="list-unstyled">
+                        {{ range .Params.categories }}
+                        {{ range first 15 (where .Site.Pages ".Section" "blog" )}}
```

Comment:
Using `hugo v0.75.1/extended` this liner errors with: 
```
Error: Error building site: failed to render pages: render of "page" failed: "/home/sumacheb/src/mattermost/developer-documentation/site/layouts/blog/single.html:73:54": execute of template failed: template: blog/single.html:73:54: executing "blog/single.html" at <.Site.Pages>: can't evaluate field Site in type string
```

Reply:
Sure @hanzei !

Reply:
![Captura de tela de 2020-10-19 22-49-18](https://user-images.githubusercontent.com/43054051/96530315-92dabf80-125d-11eb-9299-00e3764ea039.png)
Fixed and running locally :)